pkgs/development/python-modules/ssdpy/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "ssdpy";
+  version = "0.4.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-SXHGoBD3fxR+zb7JWT8tkYfD+2Nliw9ewI9Me+I4dCU=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
```

Comment:
This is missing a pythonImportsCheck

Reply:
Added the imports check!

To be able to run the tests, I had to change to fetching the sources from GitHub, as the sources from PyPI don't include the tests.

I also had to disable several tests that try to do networking. Let me know if there's a way to be able to run these that I'm not aware of.

Reply:
Disabling tests that require networking access is the general procedure around Nixpkgs.